database/config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
You'd probably want to git ignore these in the long run. Especially if requisitioning and external db.